app/uk/gov/hmrc/individualsemploymentsapi/controller/EmploymentsController.scala

```diff
@@ -22,23 +22,18 @@ import javax.inject.{Inject, Singleton}
 import org.joda.time.Interval
 import play.api.hal.Hal._
 import play.api.hal.HalLink
-import play.api.mvc.hal._
 import play.api.libs.json.Json.{obj, toJson}
-import play.api.mvc.{Action, Controller}
-import uk.gov.hmrc.domain.Nino
-import uk.gov.hmrc.individualsemploymentsapi.error.ErrorResponses.MatchNotFoundException
-import uk.gov.hmrc.individualsemploymentsapi.error.Recovery
-import uk.gov.hmrc.individualsemploymentsapi.service.{EmploymentsService, SandboxEmploymentsService}
+import play.api.mvc.hal._
+import play.api.mvc.Action
+import uk.gov.hmrc.individualsemploymentsapi.service.{EmploymentsService, LiveEmploymentsService, SandboxEmploymentsService}
 import uk.gov.hmrc.individualsemploymentsapi.util.JsonFormatters._
 
 import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
-import scala.concurrent.Future.{failed, successful}
 
-abstract class EmploymentsController(employmentsService: EmploymentsService) extends Controller with Recovery {
+abstract class EmploymentsController(employmentsService: EmploymentsService) extends CommonController {
 
-  def root(matchId: UUID) = Action.async {
-    resolve(matchId) map { _ =>
+  def root(matchId: UUID) = Action.async { implicit request =>
+    employmentsService.resolve(matchId) map { _ =>
       val payeLink = HalLink("paye", s"/individuals/employments/paye/match/$matchId{?fromDate,toDate}", title = Option("View individual's employments"))
```

Comment:
links do not match ?matchId=MATCHID...
though there is a ticket for it: OGD-426